src/tim/vx/CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+message("src/tim/vx")
```

Comment:
can we have a cmakelist for the tim/vx/internal/ and include it in the cmakelist of tim/vx

Reply:
Of course.  I'll add it.